From 2e7bfd08a306bd9e80b22097ef8fe66e1dd85054 Mon Sep 17 00:00:00 2001
From: Jeffery To <jeffery.to@gmail.com>
Date: Wed, 31 May 2023 00:00:11 +0800
Subject: [PATCH] unix-ffi/uu: Add unix-ffi version of `uu` package.

This package reuses the code from the python-stdlib version of `uu` but
requires the unix-ffi version of `os-path`.

This also updates `email.message` to require this version of `uu`.

Signed-off-by: Jeffery To <jeffery.to@gmail.com>
---
 unix-ffi/email.message/manifest.py | 2 +-
 unix-ffi/uu/manifest.py            | 6 ++++++
 2 files changed, 7 insertions(+), 1 deletion(-)
 create mode 100644 unix-ffi/uu/manifest.py

--- a/unix-ffi/email.message/manifest.py
+++ b/unix-ffi/email.message/manifest.py
@@ -1,7 +1,7 @@
 metadata(version="0.5.3")
 
 require("re", unix_ffi=True)
-require("uu")
+require("uu", unix_ffi=True)
 require("base64")
 require("binascii")
 require("email.utils", unix_ffi=True)
--- /dev/null
+++ b/unix-ffi/uu/manifest.py
@@ -0,0 +1,6 @@
+metadata(version="0.5.1")
+
+require("binascii")
+require("os-path", unix_ffi=True)
+
+module("uu.py", base_path="../../python-stdlib/uu")
